rosidl_default_generators/package.xml

```diff
@@ -9,6 +9,7 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <buildtool_export_depend>ament_cmake_core</buildtool_export_depend>
```

Comment:
vertical whitespace to match the rest of the file?

Reply:
I would interpret the current file as: keep related dependencies in block and separate unrelated dependencies by an empty line. Since `ament_cmake_core` as well as `rosidl_cmake` are "similar in that they provide CMake API I kept them together.